plugins/modules/ec2_instance.py

```diff
@@ -48,11 +48,11 @@
     type: int
   instance_type:
     description:
-      - Instance type to use for the instance, see U(https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html).
+      - Instance type to use for the instance, see
+        U(https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html).
       - Only required when instance is not already present.
-      - If not specified, C(t2.micro) will be used.
-      - In a release after 2023-01-01 the default will be removed and either I(instance_type) or
-        I(launch_template) must be specificed when launching an instance.
+      - At least one of I(instance_type) or I(launch_template) must be specificed when launching an
+        instance.
```

Comment:
Think it was because I hit a line wrap.  because this is YAML there's no effect to the docs.

The compiled docs turn this into
```
+<p>At least one of <em>instance_type</em> or <em>launch_template</em> must be specificed when launching an instance.</p>
```
(see https://github.com/ansible-collections/amazon.aws/pull/1315#issuecomment-1370640474)